velox/exec/prefixsort/PrefixSortEncoder.h

```diff
@@ -31,9 +31,10 @@ class PrefixSortEncoder {
   /// 1. Only int64_t is supported now.
   /// 2. Encoding is compatible with sorting ascending with no nulls.
   template <typename T>
-  static FOLLY_ALWAYS_INLINE void encode(T value, char* row);
+  static FOLLY_ALWAYS_INLINE void encode(bool ascending, T value, char* row);
```

Comment:
Comments need updating. Null support needs to be added.

Also, let's make sure we have dedicated unit tests for this logic. It would be helpful to extract changes to PrefixSortEncoder into a separate PR and document clearly how we encode values to allow byte-by-byte comparison compatible with sorting flags.